spec/services/anonymous_shadow_creator_spec.rb

```diff
@@ -78,6 +78,16 @@
 
       expect { AnonymousShadowCreator.get(user) }.to_not raise_error
     end
-  end
 
+    it "fallback to username 'anonymous' if the translation for 'anonymous' consist from disallowed characters" do
```

Comment:
```suggestion
    it "falls back to username 'anonymous' if the translation for 'anonymous' contains disallowed characters" do
```

Reply:
Would be "translation _consist of_ disallowed characters" correct? The idea here is that the translation contains _only_ disallowed characters. If only a part of characters is disallowed, we just strip them off.

Reply:
Good question. In that case I would say it should be "consists entirely of disallowed characters" or "consists solely of disallowed characters"

Reply:
(but to be honest I am not a stickler for language in test strings here. I'm pretty sure any dev will easily understand what you mean. It's just a suggestion!)

Reply:
"consists entirely of disallowed characters" sounds good, thank you!